tools/deprecate_version/deprecate_version.py

```diff
@@ -135,9 +137,16 @@ def GetRuntimeAlreadyTrue():
   with open('source/common/runtime/runtime_features.cc', 'r') as features:
     for line in features.readlines():
       match = runtime_features.match(line)
-      if match and 'test_feature_true' not in match.group(1):
-        print("Found existing flag " + match.group(1))
-        runtime_already_true.append(match.group(1))
+      # hit the sentinel value: no more true flags.
+      if not match:
+        continue
+      guard = match.group(1)
+      if guard == 'envoy.reloadable_features.test_feature_false':
+        print("Found sentinel\n")
+        return runtime_already_true
+      if 'test_feature_true' not in guard:
+        print("Found existing runtime guard " + guard)
```

Comment:
this is found existing _flag_ right? i.e the flags that are true by default.

Reply:
no, I need to stop using "flags"